src/app/core/store/projects/projects.effects.ts

```diff
@@ -128,6 +128,10 @@ export class ProjectsEffects {
         mergeMap(newProject => {
           const actions: Action[] = [new ProjectsAction.CreateSuccess({project: newProject})];
 
+          if (onSuccess) {
+            actions.push(new CommonAction.ExecuteCallback({callback: () => onSuccess(newProject)}));
```

Comment:
There is a much easier way how to do this now. Have a look at the usages of `createCallbackActions` function.